src/Avalonia.Controls/Grid.cs

```diff
@@ -1228,20 +1224,23 @@ private static LayoutTimeSizeType GetLengthTypeForRange(
         /// <summary>
         /// Distributes min size back to definition array's range.
         /// </summary>
+        /// <param name="definitions">Array of definitions to process.</param>
         /// <param name="start">Start of the range.</param>
         /// <param name="count">Number of items in the range.</param>
-        /// <param name="requestedSize">Minimum size that should "fit" into the definitions range.</param>
-        /// <param name="definitions">Definition array receiving distribution.</param>
-        /// <param name="percentReferenceSize">Size used to resolve percentages.</param>
+        /// <param name="spacing"><see cref="ColumnSpacing"/> or <see cref="RowSpacing"/></param>
+        /// <param name="desiredSize">Minimum size that should "fit" into the definitions range.</param>
         private void EnsureMinSizeInDefinitionRange(
             IReadOnlyList<DefinitionBase> definitions,
             int start,
             int count,
-            double requestedSize,
-            double percentReferenceSize)
+            double spacing,
+            double desiredSize)
```

Comment:
I was really confused at the removal of `percentReferenceSize` here (we ideally don't want to touch code that isn't related to spacing). But apparently this parameter was never used! It's the same thing in the original WPF source code.